test-app/README.md

```diff
@@ -0,0 +1,36 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
The parent folder should be called example, like in our other SDKS

Reply:
so it should be example/test-app

Reply:
`example/app-router` & `example/page-router`

this is not a test app, it's an example

Reply:
ok updated it